azure-pipelines.yml

```diff
@@ -0,0 +1,33 @@
+trigger:
+  # What branches will trigger a build?
+  branches:
+    include:
+      # Any Pull Request merging into the master branch
+      - master
+
+#Use the latest image with VS2019 on Windows Server 2019
+pool:
+  vmImage: "windows-2019"
+
+steps:
+  # Install the prereqs
+  - task: DotNetCoreCLI@2
+    inputs:
+      command: "custom"
+      custom: "tool"
+      arguments: "install --global Codecov.Tool"
+  - task: DockerInstaller@0
+    inputs:
+      dockerVersion: "17.09.0-ce"
+  # Build Tasks
+  - pwsh: |
+      .\Install-Prerequisites.ps1
+    displayName: "Install prerequisites"
+  - pwsh: |
+      .\Invoke-Tests.ps1
+    displayName: "Run tests and coverage"
+
+  - script: |
+      codecov -f coverage.json -t $(CODECOV_TOKEN)
+    displayName: Upload coverage to codecov.io
+    condition: always()
```

Comment:
I think you should keep the `script` block at the bottom of each step as it messes with syntax